libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/TypeRep.hs

```diff
@@ -930,6 +932,9 @@ genSizedRep _ VoteR = arbitrary
 
 genRep ::
   forall era b.
+  ( Arbitrary (PParamsHKD StrictMaybe era)
+  , EraPParams era
+  ) =>
```

Comment:
This function shouldn't have any constraints, you probably just have to case match on a proof somewhere

Reply:
I did this, and it got rid of some of them. Thanks! :raised_hands: